x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -70,3 +76,60 @@ export const StyledTime = memo(styled('time')`
   display: inline-block;
   text-align: start;
 `);
+
+interface StyledCopyableField {
+  readonly borderColor: string;
+  readonly hoverBackground: string;
+}
+
+const StyledCopyableField = styled.div<StyledCopyableField>`
+  border: 0.1em solid;
+  border-color: ${(props) => props.borderColor};
+  border-radius: 3px;
+  padding: 4px;
+
+  &:hover {
+    background-color: ${(props) => props.hoverBackground};
+    color: #fff;
+  }
+`;
+
+export const CopyablePanelField = memo(
+  ({ textToCopy, content }: { textToCopy: string; content: JSX.Element | string }) => {
+    const { linkColor, pillStroke } = useColors();
+    const [isOpen, setIsOpen] = useState(false);
+    const onMouseEnter = () => setIsOpen(true);
+
+    const buttonContent = (
+      <StyledCopyableField
+        borderColor={pillStroke}
+        hoverBackground={linkColor}
+        onMouseEnter={onMouseEnter}
+      >
+        {content}
+      </StyledCopyableField>
+    );
+
+    const onMouseLeave = () => setIsOpen(false);
+
+    return (
+      <div onMouseLeave={onMouseLeave}>
```

Comment:
:red_circle: for the following reason:

https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

"All functionality of the content is operable through a keyboard interface..."

Following that guidance, "Copy" links can't be tied/triggered to "hover"/"mouseEnter"/"mouseLeave" states or other states that can't be accessed with a keyboard. We could probably wire up something comparable with "focus"/"lose focus" so it can be done with keyboards although it's not as straightforward as you might think. IIRC there might be some things in EUI that help with this.

Reply:
👍 thanks! fyi @monina-n @lindseypoli

Reply:
We decided that while it is not accessible at the moment, there are plans to make timeline more accessible. Given that, in addition to not wanting to give the users a new behavior to learn or clutter the ui, we're gonna match how timeline does it for now